lib/pacemaker/pcmk_sched_clone.c

```diff
@@ -477,36 +477,37 @@ find_probed_instance_on(const pe_resource_t *clone, const pe_node_t *node)
     return NULL;
 }
 
-// For anonymous clones, only a single instance needs to be probed
+/*!
+ * \internal
+ * \brief Probe an anonymous clone on a node
+ *
+ * \param[in] clone  Anonymous clone to probe
```

Comment:
These should be `[in,out]` -- see `resource_alloc_functions_t:create_probe()`

Reply:
Thanks